tests/test_todo.py

```diff
@@ -57,29 +56,29 @@ async def test_async_setup_entry(
         assert mock_add_entities.call_count == 1
 
 
-@pytest.mark.asyncio
 async def test_create_todo_item(hass: HomeAssistant, mock_api, mock_coordinator):
     """Test creating a todo item."""
     grocery_list = MagicMock(id="grocery_list", title="Grocery List")
+    list_prefix = ""
     mock_coordinator.data = [
         {"id": "grocery_list", "title": "Grocery List", "items": []}
     ]
-    entity = GoogleKeepTodoListEntity(mock_api, mock_coordinator, grocery_list)
+    entity = GoogleKeepTodoListEntity(mock_api, mock_coordinator, grocery_list, list_prefix)
```

Comment:
Please format with [Black](https://github.com/psf/black). This line along with 81 and 107 triggers a line length warning with Ruff.

Reply:
I can add black, but as ruff is already being used, might as well try that. 
I have copied the vscode settings from homeassistant repo:
* suggested extensions
* ruff format on save
* ruff-format pre-commit hook

The latter should ensure that bad formatting isn't checked in in the future.

Reply:
That works, using Ruff is fine too. Just trying to avoid linter warnings.